site/pmguides/src/websockets_python/websockets_python.md

```diff
@@ -0,0 +1,182 @@
+
+![Inspect handshake details](./assets/expand.png)
+
+The connection we established between the Postman client and the local server is bidirectional and the server will echo your message back to you. Under the **Message** tab for the connection, write "Hello World!" and **Send**.
+
+![Message sent and received](./assets/MessageEcho.png)
+
+Above the connection message, there are now two new messages. One has an up-arrow icon next to it, showing it's outgoing, and one has a down-arrow icon, showing it's incoming. The message was sent and the echo arrived back in the same second. If you expand them, they're just the message text. 
+
+
+<!-- ------------------------ -->
+
+## Broadcasting Messages
+
+Duration: 3
+
+Let's pretend we're building an extremely simple chat room server. In this chat room, any message sent to the server will be broadcast to every connection.
+
+### Update the code
+
+Terminate the running server (**Ctrl + C**). This shuts down the server and Postman will register that the connection terminated. 
+
+Update `websockets.py` to the code below so it will now *broadcast* incoming messages to all connected clients. Save the changes.
+
+```python
+import asyncio
+import websockets
+
+CONNECTIONS = set()
+
+async def echo(websocket):
+  if websocket not in CONNECTIONS:
+    CONNECTIONS.add(websocket)
+  async for message in websocket:
+    websockets.broadcast(CONNECTIONS,message)
+ 
+async def main():
+    async with websockets.serve(echo, "localhost", 8765):
+        await asyncio.Future()  # run forever
+
+asyncio.run(main())
+```
+
+Run it.
+
+```bash
+$ python websockets.py
+```
+
+### Open a second WebSockets connection
+
+Select **New > WebSocket Request** to open a second tab and set the URL for the second tab to `ws://localhost:8765`, just like the first.
+
+In each tab, connect to the WebSockets server.
+
+### Send some messages
+
+In the first tab, try sending the message: "Hello! How are you?"
+
+In the second tab, try sending the message: "I'm well!"
+
+You'll get something like this:
+![Connections side by side](./assets/Broadcast.png)
+
+Note that the sender of the message sees both the outgoing message to the server and the incoming broadcast of what they sent. The other person or people connected to the server only see the incoming broadcast of messages they didn't send. 
+
+<!-- ------------------------ -->
+
+## Next Steps
+
+Duration: 1
+
+More and more developers are exploring [WebSocket APIs in Postman](https://learning.postman.com/docs/sending-requests/websocket/websocket/). According to Postman’s [State of the API 2022](https://www.postman.com/state-of-api/api-technologies/#api-technologies) report, WebSockets are used by 26% of respondents.
+
+![WebSockets are used by 26% of survey respondents](./assets/graph.png)
```

Comment:
This is great and I'm glad you included it! I'm not sure "Next Steps" is really an accurate heading here, though.

Reply:
That's straight from the Node version. I kept it for consistency.